js/client-battle.js

```diff
@@ -1242,13 +1244,15 @@
 			buf += '<p><label class="optlabel"><input type="checkbox" name="ignoreopp"' + (this.battle.ignoreOpponent ? ' checked' : '') + '/> Ignore Opponent</label></p>';
 			buf += '<p><strong>All battles</strong></p>';
 			buf += '<p><label class="optlabel"><input type="checkbox" name="ignorenicks"' + (Tools.prefs('ignorenicks') ? ' checked' : '') + ' /> Ignore nicknames</label></p>';
+			buf += '<p><label class="optlabel"><input type="checkbox" name="righthandbattles"' + (Tools.prefs('righthandbattles') ? ' checked' : '') + ' /> Open new battles on the right side</label></p>';
```

Comment:
As far as I can tell, if there's not enough space, it'll display the newly joined room/batte in full width. Because of that, the behavior with and without this option enabled is basically identical in that case, except for the position of the room tab in the top bar. Is it really worth disabling the option for that?